src/model/guild/guild_id.rs

```diff
@@ -108,7 +108,8 @@ impl GuildId {
     /// Create a voice channel in a guild with the name `test`:
     ///
     /// ```rust,ignore
-    /// use serenity::model::{ChannelType, GuildId};
+    /// use serenity::model::id::GuildId;
+    /// use serenity::model::channel::ChannelType
```

Comment:
Missing semicolon at the end. 
Should be `use serenity::model::channel::ChannelType;`